src/darwin/Framework/CHIP/ServerEndpoint/MTRServerEndpoint.h

```diff
@@ -14,7 +14,6 @@
  *    limitations under the License.
  */
 
-#import <Foundation/Foundation.h>
```

Comment:
I'll stop asking about this, but I think these Foundation imports need to stay in a lot of these places.